metamorph/plugins/morph_resultsdb.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/python
+import argparse
+import logging
+import logging.config
+import json
+import time
+import os
+import requests
+
+from metamorph.lib.logging_conf import setup_logging
+
```

Comment:
I don't know if it's a RH standard, but I generally separate stdlib imports, 3rd party imports and then project local imports by a blank line, such as the following:
```
import argparse
import logging
import logging.config
import json
import time
import os

import requests

from metamorph.lib.logging_conf import setup_logging
```